datasets/swda/swda.py

```diff
@@ -73,19 +82,38 @@
 _LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License"
 
 # Dataset main url.
-_URL = "https://github.com/NathanDuran/Switchboard-Corpus/raw/master/swda_data/"
+_URL = "https://github.com/cgpotts/swda/raw/master/swda.zip"
 
 
 class Swda(datasets.GeneratorBasedBuilder):
     """
-    Switchboard Dialog Act Corpus
+    This is the HuggingFace Dataset class for swda.
+
+    Switchboard Dialog Act Corpus Hugging Face Dataset class.
     The Switchboard Dialog Act Corpus (SwDA) extends the Switchboard-1 Telephone Speech Corpus, Release 2,
     with turn/utterance-level dialog-act tags. The tags summarize syntactic, semantic, and pragmatic information
     about the associated turn. The SwDA project was undertaken at UC Boulder in the late 1990s.
+
+    'swda_filename':       (str) The filename: directory/basename
+    'ptb_basename':        (str) The Treebank filename: add ".pos" for POS and ".mrg" for trees
+    'conversation_no':     (int) The conversation Id, to key into the metadata database.
+    'transcript_index':    (int) The line number of this item in the transcript (counting only utt lines).
+    'act_tag':             (list of str) The Dialog Act Tags (separated by ||| in the file).
+    'caller':              (str) A, B, @A, @B, @@A, @@B
+    'utterance_index':     (int) The encoded index of the utterance (the number in A.49, B.27, etc.)
+    'subutterance_index':  (int) Utterances can be broken across line. This gives the internal position.
+    'text':                (str) The text of the utterance
+    'pos':                 (str) The POS tagged version of the utterance, from PtbBasename+.pos
+    'trees':               (str) The tree(s) containing this utterance (separated by ||| in the file).
+    'ptb_treenumbers':     (str) The tree numbers in the PtbBasename+.mrg
```

Comment:
`damsl_act_tag ` is missing here as well ?

Reply:
got them fixed and arranged in same order everywhere for consistency